tests/acceptance/help-test.js

```diff
@@ -314,6 +314,51 @@ ember version \u001b[36m<options...>\u001b[39m' + EOL + '\
           anonymousOptions: []
         },
         {
+          name: 'clean',
+          description: 'Cleans up the project from downloads(npm/bower), flushes caches and removes build results directories(tmp/ and dist/ by default).',
```

Comment:
Would prefer that this description matches the one used in the actual clean task definition above